sources/packages/api/src/services/student/student.service.models.ts

```diff
@@ -1,14 +1,10 @@
+import { AddressDetailsModel } from "../address/address.models";
+
 /**
  * Information that must be provided
  * while creating a new student.
  */
-export interface CreateStudentInfo {
+export interface StudentInfo extends AddressDetailsModel {
```

Comment:
I believe that we should simplify the number of model mapping addresses.
Do they have any difference from the one persisted on DB?